MAUI/TextInputLayout/Custom-Font.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: post
+title: Font Customization  in .NET MAUI TextInputLayout control | Syncfusion
+description: Learn here all about Font Customization  support in Syncfusion .NET MAUI Text Input Layout (SfTextInputLayout) control and more.
+platform: maui
+control: SfTextInputLayout
+documentation: ug
+---
+
+# Font Customization  in .NET MAUI Text Input Layout (SfTextInputLayout)
+
+You can customize the appearance (size, attributes, and family) of font by setting the FontFamily, FontSize, and FontAttributes properties of the LabelStyle property.
```

Comment:
You can customize the appearance (size, attributes, and family) of the font by setting the FontFamily, FontSize, and FontAttributes properties of the LabelStyle property.

Reply:
Addressed the changes